### ISSUE TITLE
Contributing back

### ISSUE THREAD

**Issue by nevir** [ebaker355/clang-format#5]
Not clear from issues or the repo (apologies if already asked) - have you contributed this back to the clang-format source? Would love to see this in the mainline builds

**Comment by ebaker355** [comment]
@nevir No, I have not attempted to contribute back. I am not really a C++ dev, and so I'm not particularly confident in my edits. However, when I have time, I'll try it and see what happens. 😉

**Comment by davetcoleman** [comment]
+1 to contributing it back, I would really appreciate the "BreakBetweenDoWhile" feature in particular. Thanks!
